homeassistant/components/homekit/type_lights.py

```diff
@@ -55,6 +58,12 @@ def __init__(self, hass, entity_id, name, *args, **kwargs):
                 .get_characteristic(CHAR_BRIGHTNESS)
             self.char_brightness.setter_callback = self.set_brightness
             self.char_brightness.value = 0
+        if CHAR_COLOR_TEMPERATURE in self.chars:
+            self.char_color_temperature = serv_light \
+                .get_characteristic(CHAR_COLOR_TEMPERATURE)
+            self.char_color_temperature.setter_callback = \
+                self.set_color_temperature
+            self.char_color_temperature.value = 140
```

Comment:
The value doesn't really matter. It will be overridden once HomeKit is started. Previously we have used the min value for it as well. If you are concerned about the temperature range, it might be a good idea to add the following after the `setter_callback`:

```py

            min_mireds = self._hass.states[self._entity_id].attributes \
                .get(ATTR_MIN_MIREDS, 153)
            max_mireds = self._hass.states[self._entity_id].attributes \
                .get(ATTR_MAX_MIREDS, 500)
            self.char_color_temperature.override_properties({
                'minValue': min_mireds, 'maxValue': max_mireds})
            self.char_color_temperature.value = min_mireds
```
That will make sure that HomeKit tries to set only valid values for Home Assistant and all values Home Assistant tries to set are supported. You would need to add the additional import for `ATTR_MIN_MIREDS and ATTR_MAX_MIREDS` from `homeassistant.components.light`. The default values `[153, 500]` are taken from `light.__init__ -> Light.min_mireds, Light.max_mireds`